app/controllers/api/spaces_controller.rb

```diff
@@ -18,42 +18,41 @@ def show
     private
 
     def render_space(space)
-      unless params[:callback].blank?
-        render js: "#{params[:callback]}(#{space_hash(space).to_json});"
-      else
+      if params[:callback].blank?
         render json: space_hash(space)
+      else
+        render js: "#{params[:callback]}(#{space_hash(space).to_json});"
       end
     end
 
     def space_hash(space)
-      memberships = space.memberships.active.published.includes(:user, :answers)
```

Comment:
why removing the user? was it just some unnecessary leftover `user` here and in the membership_hash that just got spotted?

Reply:
yep